modules/ROOT/pages/release-notes/release-notes55.adoc

```diff
@@ -264,9 +266,9 @@ The {productname} 5.5 release includes an accompanying release of the *Comments*
 * Fixed the editor's `referrer_policy` setting not working when loading additional resources to improve bundling of the plugin and improving load times.
 * Fixed elements incorrectly left in the DOM when the editor was removed.
 
-For a list of supported languages, including professionally translated language packs, see: link:{baseurl}/configure/localization/#language[Localization options - `language`].
+For a list of supported languages, including professionally translated language packs, see: xref:configure/localization.adoc#language[Localization options - `language`].
 
-For information on the Comments plugin, see: link:{baseurl}/plugins/premium/comments/[Comments plugin].
+For information on the Comments plugin, see: xref:plugins/premium/comments/index.adoc[Comments plugin].
 
 === Export 0.1.0
```

Comment:
This include below has a conversion issue in it's `{pre-release_type}` attribute declaration which is causing it to render incorrectly:
![image](https://user-images.githubusercontent.com/1575550/160269874-49b393fd-058f-417a-9aef-98482c5b9775.png)

Reply:
done